tests/unit/ReportTitleUtilsTest.ts

```diff
@@ -0,0 +1,216 @@
+            });
+
+            const result = getTitleFieldFromRNVP(mockReportID);
+
+            expect(mockedReportUtils.getReportNameValuePairs).toHaveBeenCalledWith(mockReportID);
+            expect(result).toEqual(mockTitleField);
+        });
+
+        it('should return undefined when RNVP is undefined', () => {
+            mockedReportUtils.getReportNameValuePairs.mockReturnValue(undefined);
+
+            const result = getTitleFieldFromRNVP(mockReportID);
+
+            expect(result).toBeUndefined();
+        });
+    });
+
+    describe('updateTitleFieldToMatchPolicy', () => {
+        const mockReportID = '12345';
+        const mockTitleField: PolicyReportField = {
+            defaultValue: '',
+        } as unknown as PolicyReportField;
+
+        it('should return optimistic update when valid inputs provided', () => {
+            const mockPolicy: Policy = {
+                id: 'policy123',
+                fieldList: {
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
+                    text_title: mockTitleField,
+                },
+            } as unknown as Policy;
+
+            mockedReportUtils.getTitleReportField.mockReturnValue(mockTitleField);
+
+            const result = updateTitleFieldToMatchPolicy(mockReportID, mockPolicy);
+
+            expect(mockedReportUtils.getTitleReportField).toHaveBeenCalledWith(mockPolicy.fieldList);
+            expect(result).toHaveLength(1);
+            expect(result.at(0)).toEqual({
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${mockReportID}`,
+                value: {
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
+                    expensify_text_title: mockTitleField,
+                },
+            });
+        });
+
+        it('should return empty array when policy is undefined', () => {
+            const result = updateTitleFieldToMatchPolicy(mockReportID, undefined);
+
+            expect(result).toEqual([]);
+            expect(mockedReportUtils.getTitleReportField).not.toHaveBeenCalled();
+        });
+    });
+
+    describe('updateTitleFieldWithExactValue', () => {
+        const mockReportID = '12345';
+
+        it('should return optimistic update with exact value', () => {
+            const policyTitleField = 'Exact Title Value';
+
+            const result = updateTitleFieldWithExactValue(mockReportID, policyTitleField);
+
+            expect(result).toHaveLength(1);
+            expect(result.at(0)).toEqual({
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${mockReportID}`,
+                value: {
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
+                    expensify_text_title: policyTitleField,
+                },
+            });
+        });
```

Comment:
This whole test doesn't make any sense to me. The expensify text title report name value pair should always be an object, not an exact string. Also, how do we even plan to use the update title field with exact value function?